src/util/Util.js

```diff
@@ -174,6 +174,51 @@ class Util {
         return webpMedia;
     }
 
+    /**
+     * Formats a media to ogg/opus
+     * @param {MessageMedia} media
+     * 
+     * @returns {Promise<MessageMedia> media} in opus/ogg (audio) format
+     */
+    static async formatToOpusAudio(media) {
+        if (!media.mimetype.includes('audio'))
+            throw new Error('media is not an audio');
+
+        if (media.mimetype === 'audio/ogg; codecs=opus')
+            return media
+
+        const tempFile = path.join(
+            tmpdir(), `${Crypto.randomBytes(6).readUIntLE(0, 6).toString(36)}.ogg`
```

Comment:
Maybe use the same datadir from options (or do this in memory)?

Reply:
ffmpeg does not work great in memory, and I followed the same format as other Util functions, so I think this should stay. Makes the whole class standardized and if someone wants to change all the temp locations, it's better to do it at once.